src/primitives.jl

```diff
@@ -583,8 +583,22 @@ function ExchangeGraph(snd;neighbors=nothing,symmetric=false)
     end
 end
 
+function _is_included(snd_ids_a,snd_ids_b)
```

Comment:
Just a small detail: can you move this function inside ExchangeGraph_impl below?

If it only used there, I prefer to move it to keep the global namespace of PartitionedArrays clean of such small functions.

Reply:
Done.